_posts/_posts/0000-01-02-jwitchalls10.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Helloooooooo
```

Comment:
![lol](https://user-images.githubusercontent.com/67758859/86540414-136aef80-befd-11ea-8c4d-6a2d96ca33eb.jpg)
add this